### ISSUE TITLE
Changing colors 

### ISSUE THREAD

**Issue by fakhamatia** [finnhvman/matter#8]
How can i change default colors? 
I cant understand this color code
`.matter-warning {
    --matter-theme-rgb: var(--matter-warning-rgb, 238, 102, 0);
    --matter-ontheme-rgb: var(--matter-onwarning-rgb, 255, 255, 255);`

I want change warning because not near yellow color bootstrap
![Screenshot from 2019-04-17 23-41-33](https://user-images.githubusercontent.com/20030910/56314434-5a3ebf00-616a-11e9-83c1-112c8693c38d.png)

**Comment by willpower232** [comment]
`--matter-warning-rgb` and `--matter-onwarning-rgb` are CSS Custom Properties (or variables).

I think you could override the value without changing the code by adding this to your own code 
```
:root {
    --matter-warning-rgb: 238, 102, 0;
    --matter-onwarning-rgb: 204, 204, 204;
}
```

**Comment by fakhamatia** [comment]
Yes its work i add to main css and warning color change but the color of error class changed too .
matter-error now is yellow.

**Comment by finnhvman** [comment]
Hey @fakhamatia thanks for the issue, this was indeed a bug coming from a copy-paste error. It is now fixed in [Release 0.2.1](https://github.com/finnhvman/matter/releases/tag/0.2.1), let me know if you still have problems.

@willpower232 Thanks for explaining, this is exactly how global theming is meant to work.

Sorry for the late response, I wasn't around in the last few days.
